sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/owaspzapwrapper/scan/AbstractScan.java

```diff
@@ -354,6 +350,11 @@ protected void loadApiDefinitions() throws ClientApiException {
         }
     }
 
+    protected boolean atLeastOneURLDetected() throws ClientApiException {
```

Comment:
Please add a comment. How does that work? It is really hard to understand based on the calls to the ZAP API call.